docs/UserGuide.md

```diff
@@ -240,40 +248,49 @@ Lists all added tags.
 
 Format: `ls`
 
+Before executing command:
 ![before](images/screenshots/find_command.png)
+
+After executing command:
 ![after](images/screenshots/list_command.png)
 
 ### Undoing command : `undo`
 
 Undo a recently executed command.<br>
 You can undo these commands: `tag`, `retag`, `untag`, `label`, `unlabel`, `clear`, and `redo`.
-<div markdown="block" class="alert alert-info">
-**Warning: Command history will be deleted once the app is closed!
+<div markdown="block" class="alert alert-warning">
+**warning: Command history will be deleted once the app is closed!
```

Comment:
Missing `:`. Should be `**:warning`?

Reply:
Yeah, I miss that. Thank you for pointing it out!